### ISSUE TITLE
Server option breaks curl.js

### ISSUE THREAD

**Issue by rrrnld** [BrowserSync/grunt-browser-sync#21]
Hello,
when using the server option the socket.io.js breaks my curl.js configuration.
More explicitly, I get an error stating `Uncaught Error: Multiple anonymous defines encountered curl.js:16`

As far as I understand, it's because of the pattern used at the end of the socket.io file, which defines socket IO as an anonymous module — this leads to an error when this file is then not requested via the loader but manually inserted via script tag.
I have not found anything in the curl.js documentation but there is a similar error for require.js: http://requirejs.org/docs/errors.html#mismatch

Is there anything I can do in my code to fix the problem? If not, what would be an alternate approach of injecting socket.io that didn't break AMD-loaders?
Basically after browser_sync's injection my code looks like this:

``` html
  <script src="curl.js"></script>
…
  <script>
  curl(['foo', 'bar']).then(function(foo, bar) {
    // do whatever
  })
  </script>
  <script src="http://192.168.178.59:3000/socket.io/socket.io.js"></script>
  <script src="http://192.168.178.59:3001/browser-sync-client.min.js"></script>
</body>
```

Thanks in advance!
